src/Training.AirBnb.Clone.Backend/AirBnB.Api/Controllers/AccountsController.cs

```diff
@@ -0,0 +1,72 @@
+﻿using AirBnB.Api.Models.DTOs;
+using AirBnB.Application.Common.Identity.Services;
+using AirBnB.Domain.Entities;
+using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace AirBnB.Api.Controllers;
+
+[ApiController]
+[Route("api/[controller]")]
+public class AccountsController(IUserService userService, IMapper mapper) : ControllerBase
+{
+    [HttpGet]
+    public async ValueTask<IActionResult> Get()
+    {
+        var users = await userService.Get(user => true, true).ToListAsync();
+        var result = mapper.Map<IEnumerable<UserDto>>(users);
+        return result.Any() ? Ok(result) : NoContent();
+    }
+
+    [HttpGet("{userId:guid}")]
+    public async ValueTask<IActionResult> GetUserById(
+        [FromRoute] Guid userId, 
+        CancellationToken cancellationToken)
+    {
+        var user = await userService.GetByIdAsync(userId, true, cancellationToken);
+        return user is not null ? Ok(mapper.Map<UserDto>(user)) : NotFound();
+    }
+    
+    [HttpPost]
+    public async ValueTask<IActionResult> Create(
+        [FromBody] UserDto userDto, 
+        CancellationToken cancellationToken)
+    {
+        var user = mapper.Map<User>(userDto);
+        var result = await userService.CreateAsync(user, true, cancellationToken);
```

Comment:
You can skip saveChanges for `Create`, `Update` and `Delete` method as well, just as cancellation token as 

`Method(model, cancellationToken: cancellationToken);`